src/kpro_scram.erl

```diff
@@ -67,7 +67,7 @@ parse(#{ sha := Sha
   <<MyNonce:?MY_NONCE_LEN/binary, _/binary>> = ServerNonce,
   Salt = base64:decode(Salt0),
   SaltedPassword = hi(Sha, Password, Salt, Iterations),
-  FinalNoProof = bin([?CHANNEL_BINDING, ",r=", Salt]),
+  FinalNoProof = bin([?CHANNEL_BINDING, ",r=", ServerNonce]),
```

Comment:
here is an example from the widely used franz-go kafka client:

https://github.com/twmb/franz-go/blob/master/pkg/sasl/scram/scram.go#L221-L223